plugins/kernel_v2/config/disk.rb

```diff
@@ -10,7 +10,7 @@ class VagrantConfigDisk < Vagrant.plugin("2", :config)
       # Config class for a given Disk
       #-------------------------------------------------------------------
 
-      DEFAULT_DISK_TYPES = [:disk, :dvd, :floppy].freeze
+      DEFAULT_DISK_TYPES = [:disk, :dvd, :floppy].map(&:freeze).freeze
```

Comment:
The symbols themselves don't need to be frozen as they already are.

Reply:
Oooh alright, makes sense! I thought maybe we had just missed these.